CHANGE_LIMITATIONS.md

```diff
@@ -0,0 +1,21 @@
+# What changes we can't accept
+
+We wish we could accept all changes, especially changes that make the organization or readability of the project better. Unfortunately we can't, due to technical limitations 😢.
+
+## What is up with this weird module map, or strange import statement 🤷‍♂️?
+
+We host our source code inside Google and treat that as the “source of truth”. The internal build toolchain and dependency manager are not cocoapods. In order to be able to work with both internal and external tooling, we’ve had to make some concessions with how the project is set up. There are a number of considerations you must make if you’re thinking about changing things that look non-standard. 
```

Comment:
capitalize CocoaPods

Reply:
Change "how the project is set up" > "how the project is configured"

Reply:
👍